index.html

```diff
@@ -32,22 +33,22 @@
       <div id="two" class="game-board-cards"><p>let it snow</p></div>
       <div id="three" class="game-board-cards"><p>let it snow</p></div>
       <div id="four" class="game-board-cards"><p>let it snow</p></div>
-      <div id="five" class="game-board-cards"><p>100</p></div>
-      <div id="six" class="game-board-cards"><p>100</p></div>
-      <div id="seven" class="game-board-cards"><p>100</p></div>
-      <div id="eight" class="game-board-cards"><p>100</p></div>
-      <div id="nine" class="game-board-cards"><p>200</p></div>
-      <div id="ten" class="game-board-cards"><p>200</p></div>
-      <div id="eleven" class="game-board-cards"><p>200</p></div>
-      <div id="twelve" class="game-board-cards"><p>200</p></div>
-      <div id="thirteen" class="game-board-cards"><p>300</p></div>
-      <div id="fourteen" class="game-board-cards"><p>300</p></div>
-      <div id="fifteen" class="game-board-cards"><p>300</p></div>
-      <div id="sixteen" class="game-board-cards"><p>300</p></div>
-      <div id="seventeen" class="game-board-cards"><p>400</p></div>
-      <div id="eighteen" class="game-board-cards"><p>400</p></div>
-      <div id="nineteen" class="game-board-cards"><p>400</p></div>  
-      <div id="twenty" class="game-board-cards"><p>400</p></div>
+      <div id="five" class="game-board-cards"><p class="cards-value">??</p></div>
```

Comment:
Generally numbers aren't great for IDs...they're a little vague. What is this number 5 of? A category? A question?